### PR TITLE
fix #427 Options wording

### PR DESCRIPTION
**Pull request by benoitvidis**
cd @dbengsch comment in #427

### CODE REVIEW

File: bin/kuzzle [hunk 1]
```diff
@@ -40,16 +40,16 @@ program
   .arguments('[name]')
   .description('manage plugins')
   .option('    --list', 'list currently installed plugins')
-  .option('    --install', 'if plugin [name] is provided, install it from --version, --url or --path, otherwise, (re-)install all listed plugins')
-  .option('    --remove', 'removes plugin [name] from Kuzzle')
-  .option('    --activate', 'mark the plugin as "activated" (Kuzzle ignores deactivated plugins)')
-  .option('    --deactivate', 'mark the plugin as "deactivated" (Kuzzle ignores deactivated plugins)')
-  .option('    --importConfig <file>', 'import a configuration from a file for a given plugin')
+  .option('    --install', 'install a new plugin package from npm repository if [name] if provided or from --path or --url options')
```
Comment (dbengsch) [inline bin/kuzzle]: typo ?

Comment (dbengsch) [inline bin/kuzzle]: misses --packageVersion ?

Comment (xbill82) [inline bin/kuzzle]: As per https://github.com/kuzzleio/kuzzle/blob/develop/lib/api/core/plugins/packages/pluginPackage.js#L21 I don't see any `packageVersion` option available. AFAIK, only `version` is allowed, but not mandatory.

Comment (xbill82) [inline bin/kuzzle]: Wow, ok, sorry, I'm wrong. The option (for the CLI command) is actually `packageVersion` (translated to `version` [here](https://github.com/kuzzleio/kuzzle/blob/develop/bin/commands/plugins.js#L28)). But, still, I don't think it's mandatory.

Comment (benoitvidis) [inline bin/kuzzle]: It is not. 

* It cannot be used from a static source (directory or tarball).
* When installing from npm repository, if `--packageVersion` is not given, the last version is installed
* When installing from git, the `master` branch is taken as default.

File: bin/kuzzle [hunk 2]
```diff
@@ -40,16 +40,16 @@ program
...
+  .option('    --remove', 'remove plugin [name] from Kuzzle')
+  .option('    --activate', 'mark the plugin as "activated"')
+  .option('    --deactivate', 'mark the plugin as "deactivated"')
+  .option('    --importConfig <file>', 'import plugin [name] configuration from <file>')
   .option('    --get', 'get plugin [name] configuration stored in Kuzzle')
   .option('    --set <JSONObject>', 'merge current plugin configuration with JSONObject')
```
Comment (stafyniaksacha) [inline bin/kuzzle]: should be:
`--set <JSONObject> [name]`

Comment (benoitvidis) [inline bin/kuzzle]: Indeed. Fixed.

**Pull request merged**